cmd/cmd.go

```diff
@@ -172,3 +178,36 @@ var burn = &cobra.Command{
 		fmt.Printf("Transaction: %s\n", tx.TxID)
 	},
 }
+
+var mine = &cobra.Command{
+	Use:   "mine",
+	Short: "Starts the PegNet mining daemon",
+	Long: "Starts the PegNet mining daemon. Takes number of miners to use as an optional " +
+		"argument, if not specified it will use the default settings from " +
+		"the configuration file",
+	Args:    cobra.MaximumNArgs(1),
```

Comment:
If you make an arg validator like here: https://github.com/pegnet/pegnet/blob/develop/cmd/args.go#L81-L85, you can make sure the argument given is an integer or error out.

For example, this would validate 0 or 1 args, where the first optional arg has to be an integer (assuming you make the validator function)
```
Args: CustomArgOrderValidationBuilder(false, ArgValidatorInteger),
```